tools/format/swf/exporters/SWFLiteExporter.hx

```diff
@@ -706,6 +706,11 @@ class SWFLiteExporter {
 			
 		}
 		
+		if (tag.hasFontClass)
+		{
+			symbol.fontName = tag.fontClass;
```

Comment:
@jgranick you forgot to include this piece. this is also important because when shared fonts feature is used the name is only present here. from the swf doc:

Field | Type | Comment
-- | -- | --
FontClass | If HasFontClass, STRING | Class name of font to be loaded from another SWF andused for this text